app fixture is required

Hi! First of all thank you for building this! It's being really helpful.

I've found one issue that I think can be fixed with some more details on the `README`. If I follow strictly what we have there, write one test and run it that's what I will have:

```python
# this is my test
def test_health_check_is_alive(client):
    expected_result = {'status': 'alive'}

    response = client.get('/health-check')
    result = response.json

    assert result == expected_result
```

and below is the output

```bash
file /tests/integration/misc/test_healt_check.py, line 1
  def test_health_check_is_alive(client):
file /lib/python3.8/site-packages/pytest_chalice/fixtures.py, line 12
  @pytest.fixture
  def client(app):
E       fixture 'app' not found
```

The [`client`](https://github.com/studio3104/pytest-chalice/blob/master/pytest_chalice/fixtures.py#L13) fixture really expects another fixture called `app`. I found in the examples that I'd just need to create a [`conftest.py`](https://github.com/studio3104/pytest-chalice/blob/master/examples/simple/tests/conftest.py) with the app as fixture to solve it.

If you think it makes sense I can open a PR later to add it to the `README.rst` :smile:

--- a/README.rst
+++ b/README.rst
@@ -45,6 +45,21 @@ You can install "pytest-chalice" via `pip`_ from `PyPI`_::
 
 Usage
 -----
+`pytest-chalice` expects a fixture called `app` which will be the chalice app instance from your project. You can create it in your `conftest.py`:
+
+.. code-block:: python
+
+    import pytest
+
+    from chalice import Chalice
+
+    from project.app import app as chalice_app
+
+
+    @pytest.fixture
+    def app() -> Chalice:
+        return chalice_app
+
 
 .. code-block:: python
 

--- a/README.rst
+++ b/README.rst
@@ -45,6 +45,18 @@ You can install "pytest-chalice" via `pip`_ from `PyPI`_::
 
 Usage
 -----
+.. code-block:: python
+
+    from chalice import Chalice
+
+    app = Chalice(__name__)
+
+
+    @app.route('/')
+    def index:
+        return {'hello': 'world'}
+
+
 `pytest-chalice` expects a fixture called `app` which will be the chalice app instance from your project. You can create it in your `conftest.py`:
 
 .. code-block:: python
@@ -59,19 +71,6 @@ Usage
     @pytest.fixture
     def app() -> Chalice:
         return chalice_app
-
-
-.. code-block:: python
-
-    from chalice import Chalice
-
-    app = Chalice(__name__)
-
-
-    @app.route('/')
-    def index:
-        return {'hello': 'world'}
-
 
 .. code-block:: python
 

